io-ballerina/src/io/character/channel_string_io.bal

```diff
@@ -208,14 +208,14 @@ public function channelWriteJson(WritableChannel writableChannel, json content)
     }
 }
 
-# Write XML content to a channel.
+# Write XML content to a channel
 # ```ballerina
 # xml content = xml `<book>The Lost World</book>`;
 # io:Error? result = io:channelWriteXml(writableChannel, content);
 # ```
-# + writableChannel - A writable channel
+# + writableChannel - A writable channel. The possible inputs are `io:WritableByteChannel` or `io:WritableCharacterChannel`.
 # + content - XML content to write
-# + return - `io:Error` or else `()`
+# + return - Either an `io:Error` or the null `()` value when the writing was successful
```

Comment:
Same issue in the description

Reply:
Fixed in here https://github.com/ballerina-platform/module-ballerina-io/pull/37/commits/123095180ab4fe7865b5e69d8623895fe6b978e8